### ISSUE TITLE
Fail to install luci-theme-material_0.2.18-1_all.ipk

### ISSUE THREAD

**Issue by lacek** [LuttyYang/luci-theme-material#39]
My device is on 15.05.1, r48532.

Got the following error during installation:

```
root@OpenWrt:/tmp# opkg install /tmp/luci-theme-material_0.2.18-1_all.ipk 
Installing luci-theme-material (0.2.18-1) to root...
Collected errors:
 * satisfy_dependencies_for: Cannot satisfy the following dependencies for luci-theme-material:
 * 	libssp * 
 * opkg_install_cmd: Cannot install package luci-theme-material.
```

Probably related to apollo-ng/luci-theme-darkmatter#4

0.2.17 installs fine.

**Comment by alexsalex** [comment]
Same problem...
MiRoter Mini
OpenWRT 15.05.1, r48532

0.2.17  installs fine.

**Comment by jougs** [comment]
This looks like a duplicate of #38.

**Comment by Cereal-Killa** [comment]
I used "--force-depends" and it worked alright: opkg install ./luci-theme-material*.ipk --force-depends

I would be nice though to fix that issue so it can be installed normally.

Running LEDE Reboot 17.01.2 r3435-65eec8bd5f / LuCI lede-17.01 branch (git-17.152.82987-7f6fc16)

**Comment by LuttyYang** [comment]
I fix it issue in version 0.2.19, thanks
